py-setup-project.md

```diff
@@ -29,9 +29,9 @@ push:
     prop_path: 'Version'
 ```
 
-5. The 'Install dependencies' section is where you will do most of your CI work. Notice it installs the requirements.txt and outputs it with the `-t` parameter to the `./lib` folder. This tells pip to dump all the installed modules to the local lib folder which you will zip up along with your project using the `zip -r Flow.Launcher.Plugin.HelloWorldPython.zip . -x '*.git*'`, where you replace this `Flow.Launcher.Plugin.HelloWorldPython` with the name of your plugin.
+5. The `Install dependencies` section is where you will do most of your CI work. Notice it installs the requirements.txt and outputs it with the `-t` parameter to the `./lib` folder. This tells pip to dump all the installed modules to the local lib folder which you will zip up along with your project using the `zip -r Flow.Launcher.Plugin.HelloWorldPython.zip . -x '*.git*'`, where you replace this `Flow.Launcher.Plugin.HelloWorldPython` with the name of your plugin.
```

Comment:
I am a little bit skeptical using the code block for a section. Maybe bold or italic?

Reply:
Maybe both? Or is it too much? 
![image](https://github.com/Flow-Launcher/docs/assets/3993179/d986ccda-add4-4eae-bef6-84ebca23dc3f)

Reply:
Is it possible to have a hyperlink actually?

Reply:
It most likely is possible, it just doesn't make sense because the sections they're describing are just below them:
![image](https://github.com/Flow-Launcher/docs/assets/3993179/0b1dd302-5406-4af1-a3a0-0949a07a5ce6)
![image](https://github.com/Flow-Launcher/docs/assets/3993179/988e6f34-1f73-48cb-b16b-b1022ec3684c)

Reply:
Like a hashtag something? If click then the page just go directly to the section?

Reply:
I understand what you mean, I just don't understand why do this if the section it mentions is only 5 or 6 lines below the mention it self.
![image](https://github.com/Flow-Launcher/docs/assets/3993179/d60c2d94-a685-4aad-8b33-693ec669f707)

Reply:
It refers to the section of the GitHub workflow file it describes, not a section of the page.